crypto/evp/p_lib.c

```diff
@@ -666,6 +666,13 @@ static int unsup_alg(BIO *out, const EVP_PKEY *pkey, int indent,
 int EVP_PKEY_print_public(BIO *out, const EVP_PKEY *pkey,
                           int indent, ASN1_PCTX *pctx)
 {
+    const char *pq = OSSL_SERIALIZER_PUBKEY_TO_TEXT_PQ;
```

Comment:
What is the meaning of _PQ and _WD?

Reply:
Arrg.. propertyquery and wantdomainparams - maybe use longer names?

Reply:
Right, because they aren't long enough already. I was hitting the 80 char limit in ugly ways in other places.